scico/optimize/admm.py

```diff
@@ -158,25 +158,29 @@ class LinearSubproblemSolver(SubproblemSolver):
             :math:`\mb{x}` update step.
     """
 
-    def __init__(self, cg_kwargs: dict = {"maxiter": 100}, cg_function: str = "scico"):
+    def __init__(self, cg_kwargs: Optional[dict] = None, cg_function: str = "scico"):
         """Initialize a :class:`LinearSubproblemSolver` object.
 
         Args:
             cg_kwargs: Dictionary of arguments for CG solver. See
-                :func:`scico.solver.cg` or
-                :func:`jax.scipy.sparse.linalg.cg`, documentation,
+                documentation for :func:`scico.solver.cg` or
+                :func:`jax.scipy.sparse.linalg.cg`,
                 including how to specify a preconditioner.
```

Comment:
Docstring should mention defaults, especially because the defaults here differ from the defaults in solver.cg.

https://github.com/lanl/scico/blob/fe2a50e54318bc9f7c2d41325b2490991bbf8c64/scico/solver.py#L245-L255

Reply:
Good point. Done.